tests/test_parser.py

```diff
@@ -679,7 +680,24 @@ def test_array_type_spec(self) -> None:
                     [
                         ModularInteger("__PACKAGE__.Byte", Number(256)),
                         Array("__PACKAGE__.Bytes", ReferenceSpec("__PACKAGE__.Byte")),
-                        MessageSpec("__PACKAGE__.Foo", [Component("Byte", "Byte")]),
+                        MessageSpec(
+                            "__PACKAGE__.Foo",
+                            [
+                                Component(
+                                    "Length",
+                                    "Byte",
+                                    [
+                                        Then(
+                                            "Bytes",
+                                            UNDEFINED,
+                                            Mul(Variable("Length"), Number(8)),
+                                            BooleanTrue(),
```

Comment:
Please remove `BooleanTrue()`, as it is not needed here. Also for readability it should be always used `TRUE` or `FALSE` if a boolean literal is needed.

Reply:
Fixed by 30c636d